pkg/scaffold/rolebinding.go

```diff
@@ -34,15 +36,19 @@ func (s *RoleBinding) GetInput() (input.Input, error) {
 	return s.Input, nil
 }
 
-const roleBindingTemplate = `kind: RoleBinding
+const roleBindingTemplate = `kind: {{if .IsClusterScoped}}Cluster{{end}}RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: {{.ProjectName}}
 subjects:
 - kind: ServiceAccount
   name: {{.ProjectName}}
+{{- if .IsClusterScoped }}
+  # Replace this with the operator namespace
```

Comment:
Nit but maybe this might make things a bit clearer: 
Replace this with the namespace the operator is deployed in.